### PR TITLE
Remove leading slash from S3Path's pathWithoutSchema

### PR DESCRIPTION
**Pull request by geoffjentry**
Fixes three centaur tests: input_mirror, sizeenginefunction, if_then_…else_expressions

### CODE REVIEW

**Comment by geoffjentry** [comment]
@kshakir huh, TIL. It currently passes **most** of the centaur tests, maybe I can square that away in a separate PR as we should be running as many as possible.

What's weird is that `long_cmd` fails consistently for me but looks to be on the good list.

**Comment by geoffjentry** [comment]
actually i'll add the 3 for now

**Comment by kshakir** [comment]
> What's weird is that long_cmd fails consistently for me

Are you using the CI config files, or your own config file? `long_cmd` generates an _extremely_ long command line that approximates one of the longer gatk best practice commands. However the test line is so long that an [optional abbreviation](https://github.com/broadinstitute/cromwell/blob/develop/src/ci/resources/build_application.inc.conf#L23) setting was added and enabled-in-CI so that the test could run.

With [a bit of setup](https://github.com/broadinstitute/cromwell/issues/4725#issuecomment-472915580) one can run `src/ci/bin/testCentaurAws.sh` and it will attempt to use the CI configs locally.

**Comment by geoffjentry** [comment]
@kshakir just a standard config file. I hadn't actually looked at the error yet, but will make a note to come back to this discussion :)

File: filesystems/s3/src/main/scala/cromwell/filesystems/s3/batch/S3BatchIoCommand.scala [hunk 1]
```diff
@@ -56,11 +49,6 @@ import cromwell.filesystems.s3.S3Path
   * @tparam U Return type of the response
   */
 sealed trait S3BatchIoCommand[T, U] extends IoCommand[T] {
-  /**
-    * S3 operation to be executed by this command
-    */
-  def operation: S3Response
```
Comment (geoffjentry) [inline filesystems/s3/src/main/scala/cromwell/filesystems/s3/batch/S3BatchIoCommand.scala]: The `operation` deletions are only semi-related. They weren't being used, which I discovered while tracing through some of the code

**Comment by mcovarr** [comment]
@geoffjentry I ran into the same problem with `long_cmd` when I made the same mistake of using a non-CI config during horicromtal testing. Some lessons learned [here](https://github.com/broadinstitute/cromwell/pull/4748/files).

**Comment by geoffjentry** [comment]
@danbills ;)

Screen Shot 2019-03-19 at 9 25 40 AM

**Pull request merged**